flutter_vlc_player/lib/src/vlc_player_controller.dart

```diff
@@ -230,8 +228,9 @@ class VlcPlayerController extends ValueNotifier<VlcPlayerValue> {
             spuTracksCount: event.spuTracksCount,
             activeSpuTrack: event.activeSpuTrack,
             isPlaying: event.isPlaying,
-            playingState:
-                event.isPlaying ? PlayingState.playing : value.playingState,
+            playingState: (event.isPlaying != null)
+                ? PlayingState.playing
+                : value.playingState,
```

Comment:
This expression will be `true` even if the `event.isPlaying` is `false`.
```suggestion
            playingState: (event.isPlaying ?? false)
                ? PlayingState.playing
                : value.playingState,
```